chapters/parallel/index.md

```diff
@@ -0,0 +1,85 @@
+## Parallelization in Jina
+
+##### Learn about how Jina allows for parallelization of data processing
+
+### Feature description and expected outcome
+
+[comment]: <> (Outline)
+
+[comment]: <> (1. Parallel pods)
+
+[comment]: <> (2. Parallel peas within pod)
+
+[comment]: <> (2a. sharding)
+
+[comment]: <> (2b. parallel processing)
+
+Jina allows for parallelization of Peas and Pods. This generally leads to an increase in throughput, as the data is split across processes. However, depending on the configuration, this can also lead to 
+
+### Before you start
+
+You are expected to be familiar with the general concepts of Jina. You should also have experience with parallelism in other systems. Jina does follow the best practices and designs from other databases or search engines. However, Jina is primarily a *neural search engine*. This means some changes and adaptations were necessary. This chapter will explain these.    
```

Comment:
- `general concepts of Jina` - link to docs overview page

Reply:
Which one is that?

Reply:
Good question! @FionnD do we have a placeholder link for that section yet?

Reply:
It will be 
chapters/core/introduction/index

Reply:
Is the way I added it correct?

Reply:
Should be. I've taken a notice that I will personally check back and make sure it works once we merge the future PR.